Jamulus.pro

```diff
@@ -1,4 +1,4 @@
-VERSION = 3.8.0
+VERSION = 3.8.0dev
```

Comment:
Please ignore this change — it will be removed when this branch is rebased on top of `master`.